frontend/src/components/UserUpdateSfdsAuth.vue

```diff
@@ -158,19 +159,19 @@ export default {
     
     let sfdsAuthString = "";
     
-    if (this.$store.state.user.attributes.sfds_auth_1) {
+    if (this.$store.state.user.attributes.sfds_auth_1 && this.$store.state.user.attributes.sfds_auth_1.length>0) {
```

Comment:
For this number of checks, a temporary variable would make this a little easier to read. Just a personal preference, feel free to ignore.

```javascript
const attributes = this.$store.state.user.attributes;
if (attributes.sfds_auth_1 && attributes.sfds_auth_1.length > 0)
```